umh-core/pkg/communicator/api/v2/http/requester.go

```diff
@@ -67,6 +67,7 @@ func GetClient(insecureTLS bool) *http.Client {
 			TLSNextProto:      make(map[string]func(authority string, c *tls.Conn) http.RoundTripper),
 			TLSClientConfig: &tls.Config{
 				InsecureSkipVerify: insecureTLS,
+				MinVersion:         tls.VersionTLS10, // Allow older TLS versions
```

Comment:
shouldnt this only be activated if insecureTLS is also set?

Reply:
It is ?

	if insecureTLS && insecureHTTPClient == nil {
		// Create a custom transport with HTTP/2 disabled and insecure TLS
		transport := &http.Transport{
			ForceAttemptHTTP2: false,
			TLSNextProto:      make(map[string]func(authority string, c *tls.Conn) http.RoundTripper),
			TLSClientConfig: &tls.Config{
				InsecureSkipVerify: insecureTLS,
				MinVersion:         tls.VersionTLS10, // Allow older TLS version